channels/qa.m3u

```diff
@@ -17,3 +17,6 @@ https://alrayyan1live.azureedge.net/alrayyan1/alrayyan1.smil/chunklist_b1428000.
 https://alrayyan2live.azureedge.net/alrayyan2/alrayyan2.smil/chunklist_b1428000.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
 http://95.170.215.101/hls/m3u8/Bein-Sport-HD1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://i.imgur.com/cqdjHHR.png" group-title="Sport",beIN Sports 1 Arabic
+http://line.protv.cc:8000/live/tNxE4rdouh/ttqs53nnsu/7776.m3u8
```

Comment:
Right now this link does not work anymore
Most of the time these type of links are testlinks from a paid service and will work 24 or 48 Hours.